apps/spruce/src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaron.tsx

```diff
@@ -29,7 +29,7 @@ const BuildBaron: React.FC<Props> = ({
   );
   return (
     <>
-      {loading && <Skeleton active paragraph={{ rows: 4 }} title={false} />}
+      {loading && <ParagraphSkeleton />}
       {(data || annotation) && (
         <BuildBaronContent
```

Comment:
could we block any rendering if `loading` is true by bringing this up? + see next comment

```
if (loading) {
   return (<ParagraphSkeleton />)
}
```